### PR TITLE
Docker demo - make "demo-ready" images from existing theia-based images

### PR DESCRIPTION
**Pull request by marcdumais-work**
The goal is to extend the existing images to make "demo" images, that include a trace server and the tutorial traces. These "demo" versions of the images can be fired-up and used immediately to do some trace viewing.

To get started quickly, look at the [README](https://github.com/marcdumais-work/trace-viewer-examples/blob/4a0ce24c43ed08a1d28cd45ab05e19cc398b67e8/docker/theia-app-demo/README.md) under folder docker/theia-app-demo/

### CODE REVIEW

File: docker/theia-app-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# make the demo image(s) that extend the base image(s)
+yarn build:docker:tate-demo
+yarn build:docker:tave-demo
+```
+
+## Run the demo images
+
+### tate-demo (isolated docker network)
+
+note: since the isolated docker network is used, you may start several containers from the tate-demo image and each one will have its own, indepandant trace server.
```
Comment (bhufmann) [inline docker/theia-app-demo/README.md:28]: Note (upper case N)

**Review by bhufmann — CHANGES_REQUESTED** [review]
Looks good to me. I was able build and run the demo apps. I found some typos which I ask you to fix.

File: docker/theia-app-demo/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Resources for the "theia-app-demo" Example docker image
+
+This folder contains the resources, needed to create a "demo" Docker image, based on one of the Theia app images. The demo image includes the necessary extras to easity perform a quick demo or immediately play with the trace viewer: the Eclipse Trace Compass Trace Server and a selection of Tutorial traces.
```
[the remark this answers is not in the document]
Comment (marcdumais-work) [reply docker/theia-app-demo/README.md:3]: Strange this was not caught by my spell checker!

**Comment by marcdumais-work** [comment]
@bhufmann comments addressed

**Review by bhufmann — APPROVED** [review]
Thanks for the contribution. Looks good to me.

**Comment by marcdumais-work** [comment]
Thanks for the review!

**Pull request merged**